frontend/src/routes/+page.svelte

```diff
@@ -147,11 +147,7 @@
   })
 </script>
 
-<Head
-  title="streamchaser"
-  description="Trying to keep track of where to watch your favorite movies and TV series? Streamchaser provides the
-  definitive guide to what each streaming service has on offer right now!"
-/>
+<Head title="streamchaser" />
```

Comment:
Could this live in +layout.svelte?

Reply:
Sure, right now our index page is the only one that would be affected anyway 🤷🏽 , but i guess it helps if we make a new route.

Reply:
After testing, the one in layout is always chosen instead of the one in the component, so this cannot happen.